spec-v1-TG/CHARTER.md

```diff
@@ -0,0 +1,23 @@
+# RVM-CSI Spec Version 1 Task Group: Charter
+
+RVM-CSI (RISC-V eMbedded - Common Software Interface) aims to specify a source-level portability layer providing a simplified transition path between different microcontrollers based on RISC-V. It may define programming interfaces in multiple programming languages, derived from a single source of truth which determines common function and macro names and descriptions across all implemented languages.
+
+The RVM-CSI Version 1 TG (Task Group) is responsible for defining and ratifying the first version of the RVM-CSI specification, under the oversight of the RVM-CSI SIG.  It will also demonstrate a reference implementation of the API for an example platform.  The test harness for this reference implementation will form a compatibility test suite for other implementations.
+
+Only a C API will be considered for this version of the specification.  However, the specification is written in YAML, conforming to a schema defined by the RVM-CSI SIG.  From this YAML, parsers will auto-generate C header files and documentation.  This leaves open the option of later generating implementations in other languages from the same SSoT (Single Source of Truth).
+
+The specification will be constructed in accordance with the aims and requirements identified in the [gap analysis document](../gap-analysis.md).
+
+As a minimum, the following areas should be covered by version 1 of the run-time API:
```

Comment:
As described in https://github.com/riscv-admin/rvm-csi/blob/gap-analysis/gap-analysis.md#run-time-hal-apis, these goals maybe could be refined in three levels.

For lower level APIs, as discussed in the bi-weekly meeting, there could be standardized APIs cover the ratified priv and unpriv RISC-V ISA spec, and provide naming guidance about vendor or platform specified  APIs

Reply:
I'm thinking this is too detailed to put into the charter?  The charter needs enough detail to determine when the TG goals have been met (so needs to list the minimum areas covered) but it doesn't need to specify the what the APIs look like, this can be job of the TG.

Reply:
> I'm thinking this is too detailed to put into the charter? The charter needs enough detail to determine when the TG goals have been met (so needs to list the minimum areas covered) but it doesn't need to specify what the APIs look like, this can be job of the TG.

Ok, then it will looks good.